xAPI.md

```diff
@@ -1884,6 +1884,8 @@ Complete IRI validation is extremely difficult, so much of the burden for ensuri
 * A library SHOULD be used to construct IRIs, as opposed to string concatenation. 
 * Values SHOULD be considered to be case sensitive unless specified otherwise. 
 * Lowercase SHOULD be used to send case insensitive data.
+* Additional properties SHOULD* NOT be added to statements and other objects unless explicitly allowed and the 
+LRS MAY* reject statements and objects containing such additional properties.
```

Comment:
Note the use of "other objects" here, so the specific requirements around About and Person are just there for clarity.